docs/mendix-teamserver-api.md

```diff
@@ -0,0 +1,56 @@
+# Sigrid API for Mendix customers using the Teamserver
+
+This document describes 2 use cases to trigger Sigrid for customers that are using the central Mendix Teamserver.
+
+please note: When the customer is using the Bring Your Own Git scenario then the below does not apply. please refer to [Mendix for GitHub](mendix-github-actions.md)
+
+## Control the onboarding
+Onboarding a new Mendix system is normaly done via [addon.mendix.com](addon.mendix.com). In some cases customers would like to take onboarding in their own hands. Examples can be automation or bulk onboarding. In those cases customers can trigger an QSM Onboarding workflow that will add a new Mendix system to an existing Sigrid. By default the mainline of the Mendix system will be scanned once a day. The scan will provide both maintanability, OSH and security findings for QSM customers. AQM customers will only see maintainability results. The onboarding will generate a status email to the user specified in the payload.
+
+## On demand scanning
+Trigger a QSM run workflow for an already onboarded system. This is useful when a customer would like to trigger a new run for certain day. The trigger can be a step in an automated pipeline or just a developer that is interested in updated results for the mainline. There will be no email, the 'on demand' scan will overwrite the previous scan results in sigrid-says.com
+
+
+### 1. Trigger an QSM Onboarding workflow
+
+POST `/qsm/{customer}`
+
+#### Info
+Note that this endpoint requires a [SIGRID CI Authentication token](authentication-tokens.md) that is valid for {customer}. 
+##### Headers
+    - 'Authorization: Bearer YOUR_SIGRID_CI_Auth_TOKEN'
```

Comment:
Call the token 'YOUR_TOKEN' like in the example request (or call the token in the request 'YOUR_SIGRID_CI_Auth_TOKEN') so it's clear they are the same thing.